<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Center</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
background-color: rgb(0, 0, 0);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,5,3" columnstretch="4,1">
    <item row="1" column="0" rowspan="2">
     <widget class="QListWidget" name="listWidget_Plots">
      <property name="styleSheet">
       <string notr="true">QListWidget#listWidget_Plots{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131, 131, 131);
border-radius: 5px
}

QScrollBar:horizontal {
   border: 0px;
   background: black;
   height: 15px;
}
QScrollBar:vertical {
   border: 0px;
   background: black;
   width: 15px;
}</string>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_Feedback">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#label_Feedback{
color: rgb(131,131,131);
background-color:rgb(0,0,0)
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#838383;&quot;&gt;Feedback Window&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frame_Process">
      <property name="styleSheet">
       <string notr="true">
QFrame#frame_Process{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1" columnstretch="1,1">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color:rgb(131,131,131)</string>
         </property>
         <property name="text">
          <string>Process Functions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_Addition">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>open process list as addition</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#pushButton_Addition{
image:url(:/nSOTScanner/Pictures/overlay Addition.png);
background: black;
}

QPushButton:pressed#pushButton_Addition{
image:url(:/nSOTScanner/Pictures/overlay Addition.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_Subtract">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>open process list as subtract</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#pushButton_Subtract{
image:url(:/nSOTScanner/Pictures/overlaySubtract.png);
background: black;
}

QPushButton:pressed#pushButton_Subtract{
image:url(:/nSOTScanner/Pictures/overlaySubtract.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_Product">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>open process list as multiply</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#pushButton_Product{
image:url(:/nSOTScanner/Pictures/overlay Multiply.png);
background: black;
}

QPushButton:pressed#pushButton_Product{
image:url(:/nSOTScanner/Pictures/overlay Multiply.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_Division">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>open process list as division</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#pushButton_Division{
image:url(:/nSOTScanner/Pictures/overlay Divison.png);
background: black;
}

QPushButton:pressed#pushButton_Division{
image:url(:/nSOTScanner/Pictures/overlay Divison.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_PlotRelatedFunctions">
      <property name="styleSheet">
       <string notr="true">
QFrame#frame_PlotRelatedFunctions{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1" columnstretch="1,1">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_PlotRelated">
         <property name="styleSheet">
          <string notr="true">color:rgb(131,131,131)</string>
         </property>
         <property name="text">
          <string>Plot Functions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_AddPlotter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a new plotter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_AddPlotter {image:url(:/nSOTScanner/Pictures/lock.png)}

QPushButton#pushButton_AddPlotter{
image:url(:/nSOTScanner/Pictures/AddPlotter.png);
background: black;
}

QPushButton:pressed#pushButton_AddPlotter{
image:url(:/nSOTScanner/Pictures/AddPlotter.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_SetArea">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Set selected area for all window</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_SetArea {image:url(:/nSOTScanner/Pictures/lock.png)}

QPushButton#pushButton_SetArea{
image:url(:/nSOTScanner/Pictures/SelectArea.png);
background: black;
}

QPushButton:pressed#pushButton_SetArea{
image:url(:/nSOTScanner/Pictures/SelectArea.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_savePlot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save all data as matlab files. Select folder and data will be saved as file name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_savePlot {image:url(:/nSOTScanner/Pictures/lock.png)}

#pushButton_savePlot{
image:url(:/nSOTScanner/Pictures/save.png);
background: black;
}

QPushButton:pressed#pushButton_savePlot{
image:url(:/nSOTScanner/Pictures/save.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_SubtractOveralAverage">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Batch process all plotters with subtraction.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_SubtractOveralAverage {image:url(:/nSOTScanner/Pictures/lock.png)}

QPushButton#pushButton_SubtractOveralAverage{
image:url(:/nSOTScanner/Pictures/inverseRes.png);
background: black;
}

QPushButton:pressed#pushButton_SubtractOveralAverage{
image:url(:/nSOTScanner/Pictures/inverseRes.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_RefreshPlotters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Refresh All Plotters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_RefreshPlotters {image:url(:/nSOTScanner/Pictures/lock.png)}

#pushButton_RefreshPlotters{
image:url(:/nSOTScanner/Pictures/refresh.png);
background: black;
}

QPushButton:pressed#pushButton_RefreshPlotters{
image:url(:/nSOTScanner/Pictures/refresh.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_MultiplyAll">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Multiply all Plotters with constant&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_MultiplyAll {image:url(:/nSOTScanner/Pictures/lock.png)}

QPushButton#pushButton_MultiplyAll{
image:url(:/nSOTScanner/Pictures/Multiply.png);
background: black;
}

QPushButton:pressed#pushButton_MultiplyAll{
image:url(:/nSOTScanner/Pictures/Multiply.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_FineTune">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Fine Tune the plotters selected. Please select 2 plotters.</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled#pushButton_FineTune {image:url(:/nSOTScanner/Pictures/lock.png)}

#pushButton_FineTune{
image:url(:/nSOTScanner/Pictures/Fine Tuning.png);
background: black;
}

QPushButton:pressed#pushButton_FineTune{
image:url(:/nSOTScanner/Pictures/Fine Tuning.png);
background: black;
border: 2px rgb(131,131,131);
border-radius: 5px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
